### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by Whynot63**

### CODE REVIEW

**Review by andresokol — COMMENTED** [review]
cc @topshik

**Review by andresokol — CHANGES_REQUESTED** [review]
Please change pull request description according to the contribution guideline - add the rationale and short description of proposed changes

**Comment by topshik** [comment]
I think we need to rethink the architecture of the added feature. Let's have a call about it on Monday.

File: andresokol.work/index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
             <div class="pv3">
                 <h1 class="f-headline lh-solid">андрей соколов</h1>
                 <h1 class="f-subheadline lh-solid">andrei sokolov</h1>
-                <p>гений миллиардер плейбой филантроп</p>
+                <p>гей миллиардер плейбой филантроп</p>
```
Comment (topshik) [inline andresokol.work/index.html:31]: Let's try to use modern tools for the problem.

**Review by topshik — CHANGES_REQUESTED** [review]
Waiting for updates from your side.

**Comment by andresokol** [comment]
Closing due to inactivity

**Pull request closed without merging**